src/vm/codeman.cpp

```diff
@@ -1628,80 +1627,6 @@ BOOL EEJitManager::LoadJIT()
-    // to be used, and all subsequent NGEN compilations continue to use the new JIT.
-    //
-    // If this is a compilation process, then we don't allow specifying a fallback JIT. This is a case where, when NGEN'ing,
-    // we sometimes need to JIT some things (such as when we are NGEN'ing mscorlib). In that case, we want to use exactly
-    // the same JIT as NGEN uses. And NGEN doesn't follow the COMPlus_UseLegacyJit=1 switch -- it always uses clrjit.dll.
-    //
-    // Note that we always load and initialize the default JIT. This is to handle cases where obfuscators rely on
-    // LoadLibrary("clrjit.dll") returning the module handle of the JIT, and then they call GetProcAddress("getJit") to get
-    // the EE-JIT interface. They also do this without also calling sxsJitStartup()!
-    //
-    // In addition, for reasons related to servicing, we only use RyuJIT when the registry value UseRyuJIT (type DWORD), under
-    // key HKLM\SOFTWARE\Microsoft\.NETFramework, is set to 1. Otherwise, we fall back to JIT64. Note that if this value
-    // is set, we also must use JIT64 for all NGEN compilations as well.
-    //
-    // See the document "RyuJIT Compatibility Fallback Specification.docx" for details.
-    //
-    // For .NET Core 1.2, RyuJIT for x86 is the primary jit (clrjit.dll) and JIT32 for x86 is the fallback, legacy JIT (compatjit.dll).
-    // Thus, the COMPlus_useLegacyJit=1 mechanism has been enabled for x86 CoreCLR. This scenario does not have the UseRyuJIT
-    // registry key, nor the AppX binder mode.
-
-    bool fUseRyuJit = true;
-
-    if ((!IsCompilationProcess() || !fUseRyuJit) &&     // Use RyuJIT for all NGEN, unless we're falling back to JIT64 for everything.
-        (newJitCompiler != nullptr))    // the main JIT must successfully load before we try loading the fallback JIT
-    {
-        BOOL fUsingCompatJit = FALSE;
-
-        if (!fUseRyuJit)
-        {
-            fUsingCompatJit = TRUE;
-        }
-
-        if (!fUsingCompatJit)
-        {
-            DWORD useLegacyJit = Configuration::GetKnobBooleanValue(W("System.JIT.UseWindowsX86CoreLegacyJit"), CLRConfig::EXTERNAL_UseWindowsX86CoreLegacyJit);
-            if (useLegacyJit == 1)
-            {
-                fUsingCompatJit = TRUE;
-            }
-        }
-
-
-        if (fUsingCompatJit)
-        {
-            // Now, load the compat jit and initialize it.
-
-            LPCWSTR pwzJitName = MAKEDLLNAME_W(W("compatjit"));
-
-            // Note: if the compatjit fails to load, we ignore it, and continue to use the main JIT for
-            // everything. You can imagine a policy where if the user requests the compatjit, and we fail
-            // to load it, that we fail noisily. We don't do that currently.
-            ICorJitCompiler* fallbackICorJitCompiler;
-            g_JitLoadData.jld_id = JIT_LOAD_LEGACY;
-            LoadAndInitializeJIT(pwzJitName, &m_JITCompilerOther, &fallbackICorJitCompiler, &g_JitLoadData);
```

Comment:
It appears in Watson dumps, mainly. Yes, Watson dumps (probably) have platform, version, and other distinguishing characteristics, but it's simply easier if the numbers are "globally" unique.